include/modem/lte_lc.h

```diff
@@ -182,12 +182,20 @@ int lte_lc_power_off(void);
  */
 int lte_lc_normal(void);
 
-/** @brief Function for requesting modem to go to or disable
- * power saving mode (PSM) with default settings defined in kconfig.
+/** @brief Function for setting modem PSM parameters (RPTAU and RAT)
```

Comment:
I think RPTAU and RAT should be written out once

Reply:
I can't see that this has been addressed. The parameters also have to be documented.
```
/** @brief Function for setting the power saving mode (PSM) parameters
 * for requested periodic tracking area update (TAU) and requested active time,
 * 
 * @param tau Requested periodic TAU.
 * @param active_time Requested active time.
```

The same goes for the eDRX function.

Reply:
Sorry. Not sure what you mean by "I think RPTAU and RAT should be written out once".

Reply:
No worries, what I meant is that the abbreviations should be written in non-abbreviated form once, like `requested periodic TAU (RPTAU)`, so that the reader understands what the letters mean in this context. I guess most people using these features already knows, but always nice to be explicit about it in the docs anyway.

Reply:
That makes sense. I'll make that change.

BTW, I noticed there are no "issues" for this repo. There are a couple of features/changes that I have and wanted to get the code owners' feedback on before submitting as a PR. What's the best way to do that?

Reply:
I think the reason behind not exposing issues on this repository is to route all support issues, improvement suggestions, feature requests and general discussions to https://devzone.nordicsemi.com
Please open a ticket there if you have questions or suggestions, any feedback is appreciated.
And also, you're welcome to put up a PR here as a draft to propose/discuss specifics.

Reply:
Thank you. I'll probably do a draft PR.

Reply:
It seems that the param documentation requested by @jtguggedal is still not added?
```
 * @param tau Requested periodic TAU.
 * @param active_time Requested active time.
```
Same or eDRX.